Update configuration doc (#129)
ResultadosDigitais/hellper: README.md

|**HELLPER_OAUTH_TOKEN**|[Slack token](/docs/CONFIGURING-SLACK.md#OAuth-Access-Token) to exeucte bot user actions| --- |
|**HELLPER_SLACK_SIGNING_SECRET**|[Slack token](/docs/CONFIGURING-SLACK.md#Signing-Secret) to verify external requests| --- |
|**FILE_STORAGE**|Hellper file storage for postmortem document| `google_drive` |
|**TIMEZONE**|Timezone for Post Mortem Meeting| `America/Sao_Paulo` |
|**HELLPER_SLA_HOURS_TO_CLOSE**|Number of hours between the incident resolution and Hellper reminder to close the incident.| `168` |

## Running the Tests

1. `make test`

## Running the application

2. `make run`

## Deployment

[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)

### Setup database

- Run this command and copy the address:

`heroku config:get DATABASE_URL`

- Run this command and past it on the YOUR_DATABASE_URL:

`heroku config:set HELLPER_DSN=YOUR_DATABASE_URL`

- Import the scheema changing `YOUR_HEROKU_APP_NAME` by your application name:

`heroku pg:psql --app YOUR_HEROKU_APP_NAME < internal/model/sql/postgres/schema/hellper.sql`

> - Configure yours [environment variables](#Variables-explanation)

Reviewer: ```suggestion
- Configure your [environment variables](#Variables-explanation)
```

## Optional Setup

### Ngrok (To receive events from Slack)

> - Download Ngrok <https://ngrok.com/> and create account

Reviewer: ```suggestion
- Download Ngrok <https://ngrok.com/> and create an account
```

| **MTTS** | Mean Time To Solution | `total solutiontime` / `total incidents` |
| **MTTR** | Mean Time To Recovery | `total downtime` / `total incidents` |

### Alerts

Alerts are useful for notifying the status of incidents. Notifications can be used in different situations, such as requesting an update of the incident status or finalizing the post-mortem. For this you can use the CLI `notify` on your CronJob service.

To use the CLI you need to build the binary file:

```shell
go build -o notify cmd/notify/main.go
```

#### Example of use

```shell
SHELL=/bin/bash
BASH_ENV=/app/.env

> # At 16:30 on every day-of-week from Monday through Friday send a report to a channel with all incidents not closed

Reviewer: ```suggestion
# At 16:30 on every week-day, from Monday through Friday, it sends a report to a selected channel with all incidents not closed
```

30 16 * * 1-5 root /app/notify --type=report --to=YOUR_SLACK_CHANNEL_ID --status=all

> # At every 30th minute sends a status update request alert for all open incidents

Reviewer: ```suggestion
#  Every 30th minute it sends a status update request alert for all open incidents
```

0/30 * * * * root /app/notify --type=channels --status=open

> # At 13:30 on every day-of-week from Monday through Friday sends a post-mortem request alert for all resolved incidents

Reviewer: ```suggestion
# At 13:30 on every week-day, from Monday through Friday, sends a post-mortem request alert for all resolved incidents
```

30 13 * * 1-5 root /app/notify --type=channels --status=resolved
```

## Contributing

Thanks for being interested in contributing! We’re so glad you want to help! Please take a little bit of your time and look at our [contributing guidelines](/docs/CONTRIBUTING.md). All type of contributions are welcome, such as bug fixes, issues or feature requests.

## Code of Conduct

Everyone interacting in the Hellper project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](/docs/CODE_OF_CONDUCT.md).

> ## Need help

Reviewer: Why the "?" was removed? I believe it makes sense as the documentation is offering a solution whenever the person who's exploring the tool is faced with a question, even if it's a "how to" one.

Author: Markdownlint removed this, I did not notice :/
https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md026
